redhat-cop/infra-ansible
# Adding 'config-selinux' role

### What does this PR do?
This PR adds a "config-selinux" role to allow targeted systems to be correctly configured with a valid SELinux state.

### How should this be tested?
See the `tests` directory, and corresponding inventory. Try various values in the inventory.

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible
